NebulaModel/DataStructures/StatisticalSnapshot.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using System.IO;
+
+namespace NebulaModel.DataStructures
+{
+    public class StatisticalSnapShot
+    {
+        //List of statistical changes for each planet that happend in one gameTick
+        public List<ProductionChangeStruct>[] ProductionChangesPerFactory;
+        public long[] PowerGenRegister;
+        public long[] PowerConRegister;
+        public long[] PowerChaRegister;
+        public long[] PowerDisRegister;
```

Comment:
To make it less confusing, I have used the same names that are used in the original code, so it would be obvious at which parts it is being used. But I can change to to full names: "Generation", "Consumption","Charging","Discharging".

![image](https://user-images.githubusercontent.com/79051050/113354987-5e0c8180-9340-11eb-96b6-c785263df786.png)

Reply:
Ok yeah, I think I still prefer to have the complete word in our internal data structures

Reply:
fixed :)